docs/supervisor-notes.md

```diff
@@ -16,7 +16,11 @@ Below is the networking diagram of velero-plugin-for-vsphere in vSphere with Tan
 
 Certain resources with the **vSphere with Tanzu** Supervisor Cluster need to be created by the Supervisor Cluster and cannot be restored. The **Velero Plugin for vSphere** blocks backup and restore of these resources and will generate errors if an attempt is made to backup or restore these resources and the backup or restore will be marked as "Partially Failed".
 
-To avoid these errors, exclude the resources from your backups. The current list of blocked resources is:
+A configmap named `velero-vsphere-plugin-blocked-resources-list` will be created in the namespace where you install velero during **Velero Plugin for vSphere** installation. Exclude the resources in the configmap from your backups to avoid these errors. Edit the configmap to add or delete more resources after plugin is deployed. Users are able to delete the configmap, in this case we'll assume there is no resource to be blocked.
+
```

Comment:
Change "Edit the configmap to add or delete more resources after plugin is deployed." to the following:
```
After the plugin is deployed, users can edit the configmap to remove resources from being blocked, or add more resources to be blocked.
```